datasets/colmap.py

```diff
@@ -189,7 +248,7 @@ def setup(self, config, split):
                     img = Image.open(img_path)
                     img = img.resize(img_wh, Image.BICUBIC)
                     img = TF.to_tensor(img).permute(1, 2, 0)[...,:3]
-                    img = img.to(self.rank) if self.config.load_data_on_gpu else img.cpu()
```

Comment:
I think you may forget to pull the latest version for this one :)

Reply:
ouch